.gitignore

```diff
@@ -15,6 +17,7 @@ node_modules
 vendor
 
 # Generated files
+source/_data/scssVariables.json
```

Comment:
Move this up to the other `source/` files, not down here with os generated files.

Reply:
Moved in 4c7aedc